### ISSUE TITLE
No headings on the jobs page

### ISSUE THREAD

**Issue by cptvitamin** [w3c/validator-suite#243]
The /jobs page has a Heading level 1 for "Breadcrumbs" but it is hidden via CSS and therefore inaccessible to screen reader users.  In order for this page to satisfy [WCAG SC 2.4.1](http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) there should be at least a single Heading that identifies the main content area (and is not hidden via CSS display property).  Consider a Heading Level 1 that reads "Jobs list" or something similar.
